tests/test_buildSpatialKernelVisitor.py

```diff
@@ -8,6 +8,8 @@
 import lsst.utils.logging as logUtils
 import lsst.pex.config as pexConfig
 
+from lsst.ip.diffim import PsfMatchConfigAL
+
```

Comment:
Similarly, this import can be avoided and `ipDiffim.PsfMatchConfigAL` can be used later. And in other tests below.